barman/backup_executor.py

```diff
@@ -616,6 +617,8 @@ def backup_copy(self, backup_info):
         if parent_backup_info:
             parent_backup_manifest_path = parent_backup_info.get_backup_manifest_path()
 
+            self._raise_summarize_wal_value_from_parent_backup(parent_backup_info)
```

Comment:
I thought about doing this here because there was already a parent_backup_info. But now i am starting to think if it makes sense to move this to PostgresBackupStrategy.start_backup which is called first thing in the PostgresBackupExecutor. The only thing is that if exception is raised, the backup_info will not be saved apparently.

Reply:
I'd say to perform the check here: https://github.com/EnterpriseDB/barman/blob/master/barman/backup.py#L601-L612

That way we avoid creating a leftover backup.info (failed). We can use the backup manager to get the backup info of the parent backup there, and check if summarize WAL was enabled for the parent.

Reply:
Agreed